src/storage/txn/commands/mod.rs

```diff
@@ -162,12 +162,13 @@ impl From<PrewriteRequest> for TypedCommand<PrewriteResult> {
                 req.take_context(),
             )
         } else {
-            let is_pessimistic_lock = req.take_is_pessimistic_lock();
+            // the inconsistent naming is for compatibility with the old version :-(
```

Comment:
What does the comment mean?

Reply:
Well it's for the previous naming in kvproto. I've removed it.